tools/buildsteps/windows/arm-uwp/BuildSetup.bat

```diff
@@ -0,0 +1,12 @@
+@ECHO OFF
+
+SET cmakeGenerator=Visual Studio 14 ARM
+SET TARGET_ARCHITECTURE=x86
```

Comment:
in uwp build `AppxManifest.xml` is generated automatically by studio . For uwp the `package.appxmanifest` is required to be included in `kodi` project, so I've used `cmake` to generate it from template.